docs/preview/03-Features/01-Azure/01-service-bus.md

```diff
@@ -0,0 +1,291 @@
+﻿---
+sidebar_label: Service bus
+---
+
+# Azure Service Bus messaging
+The `Arcus.Messaging.Pumps.ServiceBus` library provides a way to process Azure Service bus messages on queues/topic subscriptions via custom routed *message handlers*, instead of interacting with the [`ServiceBusReceiver`](https://learn.microsoft.com/en-us/dotnet/api/azure.messaging.servicebus.servicebusreceiver) yourself. 
```

Comment:
```suggestion
The `Arcus.Messaging.Pumps.ServiceBus` library provides a way to process Azure Service Bus messages on queues/topic subscriptions via custom routed *message handlers*, instead of interacting with the [`ServiceBusReceiver`](https://learn.microsoft.com/en-us/dotnet/api/azure.messaging.servicebus.servicebusreceiver) yourself. 
```

Microsoft's documentation always mentions 'Azure Service **B**us', so I'd do that here as well.